python/jupyterlab_widgets/src/manager.ts

```diff
@@ -512,7 +512,7 @@ export class WidgetManager extends LabWidgetManager {
    * Load widget state from notebook metadata
    */
   async _loadFromNotebook(notebook: INotebookModel): Promise<void> {
-    const widget_md = notebook.metadata.get('widgets') as any;
+    const widget_md = notebook.getMetadata('widgets') as any;
```

Comment:
This is also Lab 4 specific. So it should be tested like

if ( this._context.model.getMetadata ) {
} else {
}